HackYourFuture-CPH/Guide-IT
# Seeding questions table

# Description

This PR is for a file to seed the questions table of the database with a set of data, for the purpose of having some data to test with and see running in development.

Fixes #35
https://github.com/HackYourFuture-CPH/Guide-IT/issues/35

# How to test?

To test this, you need to make sure you have pulled the latest version of develop branch.
In the terminal write npm run db:setup then verify questions table in database.
Then checkout branch seeding-questions-table.
In the terminal write npm run knex -- seed:run then verify that rows have appeared in database table.
Then change back to develop branch and in the terminal write npm run db:clean.
Verify that the questions table has been removed from the database.

# Checklist

- [ x] I have performed a self-review of my own code
- [ x] This PR is ready to be merged and not breaking any other functionality
